cafebabel/static/css/styles.css

```diff
@@ -1672,6 +1672,22 @@ select#author {
   margin-bottom: 2rem;
 }
 
+article p.video-wrapper {
+  position: relative;
+  padding-bottom: 56.25%;
+  height: 0;
+  margin: 0 auto 5rem;
+  max-width: none;
```

Comment:
I'm searching why this has been forced to be reset to default.
Is this for overriding `.article-page article p`, or `.article-page article .summary > p`? 
In which case it may be easier to read to make the selector more specific.
Or is it generic on purpose?

Reply:
no you are right, it's not generic on purpose. It override `.article-page article p`. I changed it to `.article-page article p.video-wrapper` then, thanks